src/Controller/Controller.php

```diff
@@ -452,7 +452,7 @@ protected function _mergeControllerVars()
     public function implementedEvents()
     {
         return [
-            'Controller.initialize' => 'beforeFilter',
+            'Controller.beforeFilter' => 'beforeFilter',
```

Comment:
Controller components also bind to `Controller.initialize` to `beforeFilter`. If this kind of change is going to be made the two class types should match.